### DISCUSSION TITLE
Containerize

### DISCUSSION THREAD

**Discussion by idjoo** [deaafrizal/cuyuniverse-forum#202] [Ideas]
bang, saya sendiri sedang dalam perjalanan belajar dan mastery menjadi seorang devops dan ingin memberi saran untuk "meng-containerize" aplikasi ini dan dideploy menggunakan docker-compose apakah memungkinkan untuk contirbute dari sisi devops nya? dan jika memungkinkan mungkin sekalian dibikinkan ci/cd pipeline

**Comment by deaafrizal** [comment]
sangat mmemungkinkan bro, boleh coba di setup aja dlu dari segi cicd, dockfile dan hal lainnya sembari menunggu upgrade server untuk prepare kubernetes 👍

**Reply by idjoo** [reply]
siap laksanakan bang dea
